src/Recipe.js

```diff
@@ -19,25 +19,54 @@ class Recipe {
       })
       return acc;
     }, 0) / 100;
-  };
+  }
 
   displayInstructions() {
-    return this.instructions.reduce((acc, instruction) => {
+    return JSON.stringify(this.instructions.reduce((acc, instruction) => {
```

Comment:
This is a good spot for destructuring if you want to play with it more. This isn't a necessary change, but find a way to sprinkle this in if you can, since you have a lot of opportunities to implement it.

_Example:_
```javascript
const books = [
  {
    author: 'Hugh',
    title: 'No beard'
  },
  {
    author: 'Tristan',
    title: 'Beard',
  },
  {
    author: 'Joel',
    title: 'Other beard'
  }
];

books.reduce((accumulator, { author, title }) => {
  accumulator[author] = title;

  return accumulator;
}, {});
```